IO.Eventuate.Tram/Consumer/Kafka/SwimlaneDispatcher.cs

```diff
@@ -15,24 +16,34 @@ public class SwimlaneDispatcher
 
 		// The java solution used a blocking queue but none of the methods that use
 		// the blocking feature. ConcurrentQueue is simpler.
-		private readonly ConcurrentQueue<QueuedMessage> _queue =
-			new ConcurrentQueue<QueuedMessage>();
+		private readonly ConcurrentQueue<QueuedMessage> _queue = new();
 
 		// The running flag must only be accessed within a lock(_lockObject)
 		private bool _running;
+		private bool _dispatcherStopped;
+		private CancellationTokenSource _cancellationTokenSource;
+		private Task _processQueuedMessagesTask;
+		private SwimlaneDispatcherBacklog _consumerStatus;
```

Comment:
A backlog object contains a status? This may need a comment or a rename

Reply:
I think the name originated from the java code. I'll rename to _backlog